src/js/components/ServiceDetail.js

```diff
@@ -60,6 +59,15 @@ class ServiceDetail extends mixin(InternalStorageMixin, StoreMixin, TabsMixin) {
     });
   }
 
+  componentDidMount() {
+    super.componentDidMount(...arguments);
+    this.checkForVolumes();
+  }
+
+  componentWillUpdate() {
```

Comment:
In order to get this PR merged, I've added the call to `super` here. It doesn't break the tests for me. Your explanation makes sense and I am looking forward to the discussion in our next sync meeting.